components/AuctionDetails/AuctionDetails.tsx

```diff
@@ -457,6 +457,8 @@ function AuctionDetails(props: Props) {
                         <ul className={styles.list}>
                             {auctionDetails?.enchantments.map(enchantment => {
                                 let enchantmentString = <span>{enchantment.name}</span>
+                                enchantment.name=enchantment.name.replace("Ultimate_Reiterate","Duplex");
+                                enchantment.name=enchantment.name.replace("Ultimate Reiterate", "Duplex");
```

Comment:
```suggestion
                                if(enchantment.name === "Ultimate Reiterate"){
                                    enchantment.name = "Ultimate Duplex";
                                }
```

Reply:
got it!

Reply:
I checked the hypixel api and the raw output I'm getting is "Ultimate_Reiterate"
All good though the underscore should have been removed while indexing I guess